docs/en/setup/backend/ui-grafana.md

```diff
@@ -55,6 +60,27 @@ General settings:
 #### Sampled Records
 Same as the Sort Metrics.
 
+## Configure Log Dashboard
+### Dashboards Settings
+The following steps are the example of config a log dashboard:
+1. Create a dashboard named `Log`.
+2. Configure variables for the dashboard:
+<img src="./grafana-loki-variables1.jpg"/>
+3. Please make sure `service_instance` and `endpoint` variable enabled `Include All` option and set `Custom all value` to blank (typed by space button on the keyboard):
+<img src="./grafana-loki-variables2.jpg"/>
+4. `Tags` variable is a little different from others, for more details, please refer [Ad hoc filters](https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#add-ad-hoc-filters):
```

Comment:
`Ad hoc filters` markdown format seems not working well. 

<img width="896" alt="image" src="https://github.com/apache/skywalking/assets/5441976/6773ed4a-7593-4d63-a9d4-c513e65c0379">

It should be a link. Maybe the `:` breaks it? Please recheck.
The preview is on GitHub preview page. https://github.com/apache/skywalking/blob/e519edc71c8635f8a3dde9fdffc7af8e8b3da6d2/docs/en/setup/backend/ui-grafana.md

Reply:
I adjust it in many ways, it fails. Seems github preview page cant not rendering a link in markdown list. It rendering well in idea and other markdown editor.